distribution/docs/src/main/resources/content/_transformers/xformers-intro.adoc

```diff
@@ -75,7 +75,7 @@ It is recommend not to use international character sets, as they may not be disp
 Transformers are used to alter the format of a resource or its metadata to or from the catalog’s metacard format.
 
 .[[_types_of_transformers]]Types of Transformers
-<<_available_input_transformers,Input Transformers>>:: Input Transformers create metacards from input. Once converted to a Metacard, the data can be used in a variety of ways, such as in an `UpdateRequest`, `CreateResponse`, or within Catalog Endpoints or Sources. For instance, an input transformer could be used to receive and translate XML into a Metacard so that it can be placed within a `CreateRequest` in order to be ingested within the Catalog. Input transformers should be registered within the Service Registry with the interface `ddf.catalog.transform.InputTransformer` in order to notify Catalog components of any new transformers.
+<<{integrating-prefix}available_input_transformers,Input Transformers>>:: Input Transformers create metacards from input. Once converted to a Metacard, the data can be used in a variety of ways, such as in an `UpdateRequest`, `CreateResponse`, or within Catalog Endpoints or Sources. For instance, an input transformer could be used to receive and translate XML into a Metacard so that it can be placed within a `CreateRequest` in order to be ingested within the Catalog. Input transformers should be registered within the Service Registry with the interface `ddf.catalog.transform.InputTransformer` in order to notify Catalog components of any new transformers.
```

Comment:
within a `CreateRequest` ***to*** be ingested
...
`ddf.catalog.transform.InputTransformer` ***to*** notify Catalog

Reply:
fixed